codersdk/provisionerdaemons.go

```diff
@@ -282,6 +283,12 @@ type ProvisionerKey struct {
 	// HashedSecret - never include the access token in the API response
 }
 
+const (
+	ProvisionerKeyIDBuiltIn  = "11111111-1111-1111-1111-111111111111"
+	ProvisionerKeyIDUserAuth = "22222222-2222-2222-2222-222222222222"
+	ProvisionerKeyIDPSK      = "33333333-3333-3333-3333-333333333333"
+)
+
```

Comment:
These will probably need to be hardcoded on the frontend as well, since this doesn't seem to get generated.